src/js/stores/preferences.js

```diff
@@ -99,6 +99,10 @@ define(function (require, exports, module) {
             return this._preferences;
         },
 
+        get: function (key, unsetValue) {
+            return this._preferences.get(key, unsetValue);
+        },
+
```

Comment:
syntactic sugar alert: this function is a wrapper that is almost doing nothing, but the cost is that we could confused this with other data types like Immutable.Map. Besides, this is not consistent with other places that are using `getState`.